src/Silk.Data/MediatR/Infractions/GetUserInfractionsForGuild.cs

```diff
@@ -21,12 +21,11 @@ internal sealed class Handler : IRequestHandler<Request, IEnumerable<InfractionE
 
         public async Task<IEnumerable<InfractionEntity>> Handle(Request request, CancellationToken cancellationToken)
         {
-            UserEntity? user = await _db
-                                    .Users
-                                    .Include(u => u.Infractions)
-                                    .FirstOrDefaultAsync(u => u.ID == request.TargetID && u.GuildID == request.GuildID, cancellationToken);
-
-            return user?.Infractions ?? Array.Empty<InfractionEntity>().AsEnumerable();
+            var query = _db.Infractions
+                           .Where(inf => inf.TargetID == request.TargetID)
```

Comment:
Can merge the two `.Where` clauses to one: `.Where(inf => inf.TargetID == request.TargetID && inf.GuildID == request.GuildID)`

Reply:
Done just for readability, but I'd be fine to change this.